### ISSUE TITLE
ARIA attributes for global header

### ISSUE THREAD

**Issue by obulat** [cc-archive/vocabulary-legacy#727]
## Description

**'Explore CC'** button in the global header needs to have the following ARIA attributes to be accessible: 
- 'aria-haspopup' set to `true` to show that it opens up a menu when clicked
- 'aria-expanded' set to `false` when the global header menu is collapsed, and to `true` when it is expanded.
- its role should be set by 'aria-role: button'. Or, even better for accessibility, it should be a button so that we can get keyboard interaction and aria-roles for free. This will, however, require more effort in styling the element.

## Resolution

- [ ] I would be interested in resolving this bug.

**Comment by rochisha0** [comment]
I'd like to take this issue!
